pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/user/UserDto.java

```diff
@@ -29,6 +29,10 @@ public int getId() {
         return this.id;
     }
 
+    public String getIdAsString() {
```

Comment:
There was a bunch of code doing `Integer.toString(getId())` so I moved that logic here.